docs/cubeviz/export_data.rst

```diff
@@ -15,6 +15,15 @@ Spatial Regions
     :ref:`Export Spatial Regions <imviz_export>`
         Documentation on how to export spatial regions.
 
+Since Specviz can be accessed from Cubeviz, the following line of code
+can be used to extract a spatial subset named "Subset 1"::
```

Comment:
do we need to distinguish between the subset itself and the spectrum corresponding to that subset?  Maybe just "extract the spectrum of a spatial subset"?  Would it be useful to show how to access the spectrum object/mask itself, or is that more advanced functionality?

Reply:
Good call, I updated the wording to be more clear. The ticket only mentioned how to access subsets as Spectrum1D objects so that's as far as I went.